addons/wowl/static/src/main.ts

```diff
@@ -21,13 +21,15 @@ const { whenReady, loadFile } = owl.utils;
 
 (async () => {
   // prepare browser object
+  let sessionStorage: Window["sessionStorage"] = window.sessionStorage;
```

Comment:
why not owl.browser.sessionStorage?

Reply:
hmm, owl does not use it, so no i guess...